internal/server/storage/drivers/driver_zfs.go

```diff
@@ -242,8 +249,12 @@ func (d *zfs) Create() error {
 		return err
 	}
 
+	poolType, devices := d.parseSource()
+	if poolType != "" && !slices.Contains(zfsSupportedPoolTypes, poolType) {
```

Comment:
With above change, we should be able to drop the `poolType != ""` part of that as default would now be `stripe`.